### ISSUE TITLE
RuntimeExpcetion: Placing Analyzer, with deactivated ApiCulture

### ISSUE THREAD

**Issue by samtrion** [ForestryMC/ForestryMC#2203]
https://paste.dimdev.org/icosaparek.mccrash

**Comment by temp1011** [comment]
It's because honey is disabled I think. Not sure what the best way to protect against that is.

**Comment by samtrion** [comment]
Yeah, there are a lot of errors with deactivating bees. You can't use analyzer with out honey but need IT for trees
